### ISSUE TITLE
Support for vcpkg

### ISSUE THREAD

**Issue by ArthurGiannotta** [kunitoki/LuaBridge3#11]
The latest LuaBridge2 supports using vcpkg (https://vcpkg.info/port/luabridge).

Are there any plans to also include luabridge3?

**Comment by kunitoki** [comment]
Yeah nice idea, need to only understand what's needed to make a package. What about conan?

**Comment by kunitoki** [comment]
PR is being opened https://github.com/microsoft/vcpkg/pull/26600

**Comment by kunitoki** [comment]
Merged!

Will bump vcpkg support as soon as we release new versions
